comp/core/flare/helpers/send_flare.go

```diff
@@ -253,3 +254,10 @@ func SendTo(cfg pkgconfigmodel.Reader, archivePath, caseID, email, apiKey, url s
 	defer r.Body.Close()
 	return analyzeResponse(r, apiKey)
 }
+
+// GetFlareEndpoint creates the flare endpoint URL
+func GetFlareEndpoint(cfg config.Reader) string {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏I guess this function should be used somewhere in the flare ? (ie. reuse it as much as possible to avoid duplication)

Reply:
I think but maybe we need to do it in an other PR, since it is not the main goal of this one ?

Reply:
I put a card in backlog for this one